content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -117,12 +121,74 @@ kind: KubeletConfiguration
 address: "192.168.0.8"
 ```
 
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+address: "192.168.0.8"
+```
+Additionally, you can change the slices/lists values of the kubelet configuration. For example, you can override the `clusterDNS` and `featureGates` fields as follows:
```

Comment:
please give an example with overriding values

Reply:
example: `failSwapOn ` - disabled explicitly in main config and enabled in child config. May be a common scenario to enable swap via drop-in config

Reply:
This is covered in the first example.